panc/src/test/java/org/quattor/pan/PanTest.java

```diff
@@ -0,0 +1,95 @@
+package org.quattor.pan;
+
+import org.quattor.pan.exceptions.SyntaxException;
+import java.util.*;
+
+/**
+ * Created by iliclaey.
+ *
+ * When building the panc-project a panc-MAJOR.MINOR-SNAPSHOT-fat-jar-with-tests.jar file is created.
+ * This jar can be used to execute pan test files and should be used as follows:
+ * `java -Dpanc.testdir="path" -Dpanc.tmpdir="path" -jar path/to/panc-MAJOR.MINOR-SNAPSHOT-fat-jar-with-tests.jar`.
```

Comment:
what are the exitcodes on succes, failure and error (like syntax exception)?
can you try to run some tests and check the `$?` value?